coordinator/core/marbleapi.go

```diff
@@ -88,6 +90,24 @@ func (c *Core) Activate(ctx context.Context, req *rpc.ActivationReq) (*rpc.Activ
 	}
 
 	marble := c.manifest.Marbles[req.GetMarbleType()] // existence has been checked in verifyManifestRequirement
+	// add TTLS config to Env
```

Comment:
Add unit test(s). You may add a simple case to the existing TestActivate and a new test for the separated function where you can test some more cases.